### ISSUE TITLE
Sticker erasure

### ISSUE THREAD

**Issue by PoyndexterGroyper** [zoomerdev/dbased#2]
Hey bud, I noticed that every now and then all the extra stickers would disappear. I'm not sure what you use to save the stickers, but I assume it has something to do with cookies or site data, since the issue arises whenever your dlive session expires and it logs you out.

**Comment by zoomerdev** [comment]
Thanks for the report, I've got the problem fixed locally, just need to make a release

**Comment by zoomerdev** [comment]
sorry folks I know this is affecting a lot of you. Working on an update have been busy with dlive-superchats. 

Here's how you can backup your stickers in the meantime (F12 to open the browser console)

copy all stored stickers:
```
copy(localStorage['extra-stickers'])
```

restore saved stickers:
```
localStorage['extra-stickers'] = <paste here>
```

**Comment by zoomerdev** [comment]
Fixed in 1.0.2
